### ISSUE TITLE
Cannot able to login via user mobile number

### ISSUE THREAD

**Issue by Harsha18** [softkoash-ashishjain/brunchery_v1#8]

**Comment by softventure** [comment]
Is this an issue? I was able to login using the mobile number on the softkoashsolution.in.

**Comment by BhavanaSJain** [comment]
I am also not able to login via my mobile no. My login details are:
1. Mobile no:8888889562
2. Mail id: bhavana@softkoash.com
3. Password: a1a1

Able to login via mail but not through mobile no

**Comment by softventure** [comment]
Yes, this is issue in mobile app. Again Karenderia might be using some settings we need to figure out for mobile login from mobile app. I can login from web using mobile number though.

**Comment by BhavanaSJain** [comment]
![image](https://user-images.githubusercontent.com/8385702/45021311-d9a1d700-b04e-11e8-9c99-42ab2cd688f0.png)
